cmd/metacache-entries.go

```diff
@@ -169,6 +169,7 @@ func (e *metaCacheEntry) isLatestDeletemarker() bool {
 	// Fall back...
 	xlMeta, err := e.xlmeta()
 	if err != nil || len(xlMeta.versions) == 0 {
+		// FIXME: upon error is it really delete-marker?
```

Comment:
I think this is due to some legacy behavior where "if it is broken, it is deleted". Feel free to modify as you see fit.

Reply:
Okay makes sense, removing the comment.